bots/images/scripts/windows-10.bootstrap

```diff
@@ -0,0 +1,79 @@
+#! /bin/bash
+set -ex
+
+ORIGIN="$1"
+VM_NAME=win10
+
+test -e MSEdge.Win10.VMWare.zip || wget https://az792536.vo.msecnd.net/vms/VMBuild_20180425/VMWare/MSEdge/MSEdge.Win10.VMWare.zip -O MSEdge.Win10.VMWare.zip
+test -e MicrosoftWebDriver.exe || wget https://download.microsoft.com/download/F/8/A/F8AF50AB-3C3A-4BC4-8773-DC27B32988DD/MicrosoftWebDriver.exe -O MicrosoftWebDriver.exe
+test -e java-installer.exe || wget http://javadl.oracle.com/webapps/download/AutoDL?BundleId=216432 -O java-installer.exe
```

Comment:
How stable are these URLs? Does this perhaps need to read and parse an index page with a stable URL?

Reply:
I'll go over them again. Most of the urls are from the windows-8 bootstrap script.

Reply:
Oh wow, indeed - so that answers my question about "how stable", but also raises some question how old this Java installer already is :)

Reply:
So that java url is the same as windows-8 bootstrap. 

Not sure how I can get the windows 10 vm url more stable, it would require actually filling out a small form, see https://developer.microsoft.com/en-us/microsoft-edge/tools/vms/. The windows 8 ones seem stable enough though?

As soon as the vm's url would change because of new versions or something, the webdriver url would need updating also since the build number needs to match. Not sure what's the best approach here.